readme.md

```diff
@@ -64,7 +65,8 @@ Configuration:
                                  Chromium version 66.0 or later. By default, any detected Chrome Canary or Chrome (stable) will be launched.
                                                                                                                                        [default: ""]
   --port                         The port to use for the debugging protocol. Use 0 for a random port                                    [default: 0]
-  --preset                       Use a built-in configuration.                                            [choices: "full", "perf", "mixed-content"]
+  --preset                       Use a built-in configuration.                                            [choices: "full", "perf", "mixed-content", "lr-desktop", "lr-mobile"]
```

Comment:
`lr-desktop` and `lr-mobile` are really only meant for configuring runs for PSI and are subject to change at any time, so I don't think we want to expose them as an option here

Reply:
Ok reverted that.